dcpy/models/product/dataset/metadata.py

```diff
@@ -135,16 +143,27 @@ def get_columns(self, metadata: Metadata) -> list[Column]:
         return cols
 
 
+class ZipFile(File, extra="forbid"):
+    type: Literal["Zip"] = "Zip"
+    contains: list[str]
+
+
 class Package(BaseModel, extra="forbid"):
     dataset_files: list[DatasetFile]
     attachments: list[str]
+    zip_files: list[ZipFile] = []
```

Comment:
I'm still a tad unsure of the placement here - not that it doesn't work, but that it's a little unintuitive given that the rest of the `Package` is a very literal description of what is present and this is more an instruction.

Could you maybe at least add a comment here in code describing what this field is?

Reply:
Yeah, I agree - this does feel a little misplaced. Package has so far been used to describe outputs of a build, whereas these are more like instructions for... err, "packaging". I almost feel like this class should just be `BuildArtifacts`, and then we'd want a separate class for `Package`, which is where `zip_files` as above makes sense. thoughts @fvankrieken

Reply:
although... attachments aren't BuildArtifacts. Clearly need to think harder about this, but good fodder for whiteboarding tomorrow.

Reply:
Maybe what we're looking for is `Assembly` which would have a zip_files entry? 🤔